packages/header-parser/src/index.ts

```diff
@@ -289,7 +289,7 @@ export function validatePackageJson(
 
 export function getTypesVersions(dirPath: string): readonly TypeScriptVersion[] {
   return mapDefined(fs.readdirSync(dirPath), (name) => {
-    if (name === "tsconfig.json" || name === "tslint.json" || name === "tsutils") {
+    if (name === "tsconfig.json" || name === "tsutils") {
```

Comment:
Curiosity: why is `"tsutils"` special cased here?

Reply:
Absolutely no clue, surely this is not needed.

Reply:
> surely this is not needed

😰

Reply:
It sure looks like it's to allow types/tsutils and not error saying that it must be types/tsX.Y, even though that doesn't make sense at the types/ level.

Reply:
#871